frontend/src/components/core/Button.tsx

```diff
@@ -22,8 +23,8 @@ export const DefaultButton = styled.button`
 
 export const PrimaryButton = styled(DefaultButton)<any>`
   font-size: ${({ theme }) => theme.fontSize.mediumLarge};
-  font-weight: 400;
-  background: ${({ theme }) => theme.colors.gradients.blue};
+  font-weight: ${({ fontWeight }) => (fontWeight || 400)};
+  background: ${({ color }) => color};
```

Comment:
This might be personal preference, but I feel like if we have multiple styling options it might be good to have multiple containers like so: 

```jsx
export const PrimaryButton = styled(DefaultButton)`
  ...
`;

export const RedButton /* or DangerButton like Bootstrap does it */ = styled(PrimaryButton)`
  color: red;
`;

...
```

Mainly modeling this idea off of how Bootstrap does it, and also potentially to save on lines in the other files (so we don't have to specify a bunch of options everywhere or manually import ThemeConfig outside of the withTheme thing)

Reply:
Agreed, that makes sense - changed back 👍

Reply:
Agreed - done ✅